e2e/cypress/integration/main.spec.js

```diff
@@ -66,22 +66,23 @@ describe('Basic flow', () => {
         goToLabeling('Find narrowings');
         matSelect('[data-cy=tags]', 'Narrowing (lenghtwise)');
         cy.get('[data-cy=chain-tool]').click();
+
+        cy.get('[data-cy=spinner]').should('not.exist');
  
         // Enter first Chain element
-        cy.get('canvas').click(100, 100);
-        cy.get('canvas').click(200, 100);
-        cy.get('canvas').click(200, 200);
-        // NOTE: Cannot use right click event, so as a work around we are changing Tool
-        cy.get('[data-cy=chain-tool]').click();
+        cy.get('canvas').moveAndClick(300, 300);
+        cy.get('canvas').moveAndClick(200, 300);
+        cy.get('canvas').moveAndClick(300, 200);
+        // Click on first point makes loop and completes chain
+        cy.get('canvas').moveAndClick(300, 300);
  
         // Enter second Chain element
-        cy.get('canvas').click(300, 300);
-        cy.get('canvas').click(200, 300);
-        cy.get('canvas').click(300, 200);
-        cy.get('canvas').click(300, 300);
-        // NOTE: Cannot use right click event, so as a work around we are changing Tool
-        cy.get('[data-cy=chain-tool]').click();
- 
+        cy.get('canvas').moveAndClick(100, 100);
+        cy.get('canvas').moveAndClick(200, 100);
+        cy.get('canvas').moveAndClick(200, 200);
+        // NOTE: cannot simulate right click to stop drawing chain
```

Comment:
Nitpicking, but can't we leave this comment in one line ?

Reply:
Done